src/ui/common_components/ExpressionTable.tsx

```diff
@@ -88,6 +99,45 @@ class ExpressionTable extends React.Component<
       });
     }
     this.setState(new EmptyState());
+    this.editInput = undefined;
+  }
+
+  public validateEdit(): boolean {
+    if (!this.state.editMode || !this.editInput || !this.state.id) return false;
+    const inputTrim = this.state.expressionInput.trim();
+    if (!inputTrim) {
+      return this.setInvalid(browser.i18n.getMessage('inputErrorEmpty'));
+    }
+    if (inputTrim.startsWith('/') && inputTrim.endsWith('/')) {
```

Comment:
Hmm this seems like something you should probably unit test to verify the results.

Reply:
Moved to a function in LIbs to allow validation from List of Expressions (new input) as well.